searchcore/src/vespa/searchcore/proton/matching/matcher.cpp

```diff
@@ -19,6 +19,7 @@
 LOG_SETUP(".proton.matching.matcher");
 
 using search::fef::Properties;
+using namespace search::fef::indexproperties;
 using namespace search::fef::indexproperties::matching;
```

Comment:
Only import what you need using namespace search::fef::indexproperties::hitcollector.